shared/feature/networkuser/src/main/java/com/epmedu/animeal/networkuser/data/repository/AuthenticationTypeRepository.kt

```diff
@@ -0,0 +1,12 @@
+package com.epmedu.animeal.networkuser.data.repository
```

Comment:
Since domain layer should not know about other layers, repository interfaces should be in domain